app/Student.php

```diff
@@ -150,12 +150,12 @@ public function isAdmin(): bool
 
     public function isTeacher(): bool
     {
-        return $this->getUserLevel() === 1 || $this->isAdmin();
+        return $this->getUserLevel() === 1;
     }
 
     public function isStudent(): bool
     {
-        return $this->getUserLevel() === 0 || $this->isAdmin();
+        return $this->getUserLevel() === 0;
     }
```

Comment:
Just wondering, is this removal necessary for the functionality? Currently it helps by not needing to re-login with another account to test functionality of that role as an admin. If it is necessary, then just ignore this.

Reply:
This is because I get all the students in the dropdown by IsStudent(). Otherwise I see the admins in the dropdown too.

Reply:
Ah, of course 😅